convert2rhel/unit_tests/actions/backup_system_test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import os
+import unittest
+
+import pytest
+import six
+
+from convert2rhel import actions, backup, redhatrelease, repo, unit_tests
+from convert2rhel.actions import backup_system
+from convert2rhel.backup import RestorableFile
+
+
+six.add_move(six.MovedModule("mock", "mock", "unittest.mock"))
+from six.moves import mock
+
+
+OS_RELEASE_FILEPATH = "/etc/os-release"
```

Comment:
You can import that constant from `redhatrelease` instead of copying it over here.